sdk/servicebus/azure-servicebus/azure/servicebus/management/__init__.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._management_client import ServiceBusManagementClient
+from ._shared_key_policy import ServiceBusSharedKeyCredentialPolicy
+from ._generated.models import QueueDescription, AuthorizationRule, MessageCountDetails, QueueRuntimeInfo, \
+    AccessRights, EntityAvailabilityStatus, EntityStatus
```

Comment:
We don't expose generated models directly in hand-wrapped SDKs - mostly for documentation reasons, but they can also be cumbersome to use.
It's probably fine for a first preview though.

Reply:
To follow up to get convenience layer models.

Reply:
I added convenience layer classes for QueueDescription and QueueRuntime. Other small classes are very simple so I think it's not a problem to use generated. Thoughts?

Reply:
No we should not expose generated models at all.
Though as I mentioned - happy for this to be fixed in a subsequent preview.